data/schema.sql

```diff
@@ -33,7 +33,8 @@ CREATE TABLE wof_neighbourhood (
   inception DATE NOT NULL DEFAULT '0001-01-01',
   cessation DATE NOT NULL DEFAULT '9999-12-31',
   is_visible BOOLEAN NOT NULL DEFAULT true,
-  l10n_name HSTORE
+  l10n_name HSTORE,
+  wikidata TEXT
```

Comment:
nit: `wikidata_id`?

Reply:
Could leave as wikidata here, might make import easier. But the ETL should transform it to `wikidata_id` on export.